documentation/docs/getting-started/index.md

```diff
@@ -5,149 +5,121 @@ path: /getting-started/
 
 # Getting started
 
-Fusion.js is a web application framework for building high quality universal applications. It provides a modular architecture with a strong focus on testability and maintainability.
+Fusion.js, Uber’s open source universal web framework, is designed to make web development easier and produce lightweight,
+high-performing apps. We initially [built Fusion.js](https://eng.uber.com/fusionjs/) to revamp our own websites, and have
+since offered it to the community as an open source project.
 
-Here are the features you'll find in Fusion.js:
```

Comment:
Does this list live elsewhere?  I think it's still useful as an "at a glance, here are the things you can expect from Fusion.js versus other solutions".

Reply:
Maybe the link to the framework comparison page could stay, and the list of features should be in the introduction on that page.

Reply:
Peter was pretty hell bent on just removing this information dump for now. But we could bring it back in some capacity in the future. The overall messaging is still being worked on but I think the general trend is to reduce info dumps like this.

The home page does have some similar sounding blurbs to this at the very bottom so they do exist in some fashion elsewhere.

Reply:
Fair enough.  FWIW, I think we could do this in a more digestible fashion.  For example, [Electrode](http://www.electrode.io/site/web.html) does a decent job of highlighting some top-level features along with the use cases that they serve:

![image](https://user-images.githubusercontent.com/3497835/55660158-c22ff580-57b9-11e9-8168-47e19caa8a61.png)